src/client.js

```diff
@@ -81,7 +83,7 @@ export class Saturn {
       }
     }
 
-    const origins = options.origins
+    const origins = options.origins || [options.cdnURL]
```

Comment:
```suggestion
    const origins = options.origins ?? [options.cdnURL]
```

Reply:
@DiegoRBaquero updated this with your suggestion and eric's comment above